networks.py

```diff
@@ -44,6 +44,12 @@ def sample(self):
     def log_prob(self, input):
         return (-math.log(math.sqrt(2 * math.pi)) - torch.log(self.sigma) - ((input - self.mu) ** 2) / (2 * self.sigma ** 2)).sum()
 
+def enable_dropout(model):
```

Comment:
can this be a class method in MLP_Dropout?

Reply:
yep, good point.